app/src/main/res/layout/profile_activity.xml

```diff
@@ -1,44 +1,214 @@
+
+                <ImageView
+                    android:id="@+id/blurred_consumer_pic"
+                    android:layout_width="0dp"
+                    android:layout_height="174dp"
+                    android:layout_marginStart="4dp"
+                    android:layout_marginLeft="4dp"
+                    android:layout_marginEnd="8dp"
+                    android:layout_marginRight="8dp"
+                    android:layout_marginBottom="8dp"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintHorizontal_bias="1.0"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintVertical_bias="0.003"
+                    app:srcCompat="@android:drawable/screen_background_light_transparent" />
+
+                <ImageView
+                    android:id="@+id/iv_prof_pic"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="8dp"
+                    android:layout_marginLeft="8dp"
+                    android:layout_marginEnd="8dp"
+                    android:layout_marginRight="8dp"
+                    app:layout_constraintBottom_toBottomOf="@+id/blurred_consumer_pic"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintHorizontal_bias="0.492"
+                    app:layout_constraintStart_toStartOf="@+id/blurred_consumer_pic"
+                    app:layout_constraintTop_toTopOf="@+id/blurred_consumer_pic"
+                    app:layout_constraintVertical_bias="0.252"
+                    app:srcCompat="@drawable/instagram_user_outline_24" />
+
+                <TextView
+                    android:id="@+id/tv_name"
+                    android:layout_width="wrap_content"
+                    android:layout_height="27dp"
+                    android:layout_marginStart="8dp"
+                    android:layout_marginLeft="8dp"
+                    android:layout_marginTop="8dp"
+                    android:layout_marginEnd="8dp"
+                    android:layout_marginRight="8dp"
+                    android:layout_marginBottom="8dp"
+                    android:text="TextView"
+                    app:layout_constraintBottom_toBottomOf="@+id/blurred_consumer_pic"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/iv_prof_pic"
+                    app:layout_constraintVertical_bias="0.196" />
+            </androidx.constraintlayout.widget.ConstraintLayout>
+        </ViewFlipper>
+    </FrameLayout>
+
+    <com.google.android.material.navigation.NavigationView
```

Comment:
if you're using this view in multiple activities (which i think you guys are), put it in its own xml and then you can reference it with <include>

https://developer.android.com/training/improving-layouts/reusing-layouts